index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
+    integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+  <div class="container text-center my-4 py-4">
+    <h1 id="title">Survey Form</h1>
+    <p id="description">Please Fill the forms below. This is just to have some informations about you.</p>
+    <form action="" id="survey-form" class="text-left px-4">
+      <label for="name" id="name-label">Name</label><br>
+      <input class="form-control form-control-lg" type="text" id="name" placeholder="enter the name" required><br>
+      <label for="email" id="email-label">Email</label><br>
+      <input class="form-control form-control-lg" type="email" id="email" placeholder="enter the email" required><br>
+      <label for="number" id="number-label">Age</label><br>
+      <input class="form-control form-control-lg" type="number" id="number" placeholder="enter the age" min="13"
+        max="100">
+        <div class="form-group mt-4">
+          <label for="dropdown" id="dropdown-label">Which option best describes your current role?</label>
+          <select class="form-control form-control-lg" name="" id="dropdown">
+            <option>Student</option>
+            <option>Full Tiem Job</option>
+          </select>
+        </div>
+        <div class="form-group mt-4">
+          <label for="">Would you recommend freeCodeCamp to a friend?</label><br>
```

Comment:
The use of semantic HTML makes web pages more informative and adaptable, allowing browsers and search engines to better interpret content.

Kindly ensure that this entire section is in a <section> tag, this will make your code more semantic.